ports/qemu/boards/MICROBIT.mk

```diff
@@ -11,3 +11,6 @@ LDSCRIPT = mcu/arm/nrf51.ld
 SRC_BOARD_O = shared/runtime/gchelper_native.o shared/runtime/gchelper_thumb1.o
 
 MPY_CROSS_FLAGS += -march=armv7m
+
+# Inline assembler tests are disabled for this board, so exclude them.
+RUN_TESTS_ARGS = --exclude 'inlineasm|qemu/asm_test'
```

Comment:
Doesn't `run-tests.py` now include logic to skip these (or at least the `inlineasm`) tests if the target doesn't support them?

Reply:
Missed this during the last rebase, thanks for noticing!  This should be fixed now.